packages/create-remix/templates/arc-stack/app/models/user.ts

```diff
@@ -1,6 +1,11 @@
 import arc from "@architect/functions";
 import bcrypt from "bcryptjs";
 
+function cleanUser(user: any) {
```

Comment:
Do we still need this for arc? Can we take the same approach here that we do for fly?

Reply:
won't be needed after i do that